src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Season.Add": "Ajouter une nouvelle saison",
+  "FSC.Configuration.Month.Title": "Paramètres du mois",
+  "FSC.Configuration.Month.Name": "Nom du mois",
+  "FSC.Configuration.Month.NameHelp": "Le nom textuel de ce mois.",
+  "FSC.Configuration.Month.Days": "Nombre de jours",
+  "FSC.Configuration.Month.DaysHelp": "Le nombre de jours qui composent ce mois.",
+  "FSC.Configuration.Month.Intercalaire": "Mois Intercalaire",
+  "FSC.Configuration.Month.IntercalaryHelp": "Si ce mois doit être considéré comme un mois intercalaire ou non.",
+  "FSC.Configuration.Month.IntercalaryInclude": "Inclure le mois intercalaire dans les calculs de jour",
+  "FSC.Configuration.Month.IntercalaryIncludeHelp": "Si coché, le mois intercalaire sera traité comme un mois normal pour calculer le jour de la semaine et le nombre total de jours dans une année, mais sera toujours répertorié comme un mois intercalaire.",
+  "FSC.Configuration.Month.DayOffset": "Décalage du jour",
+  "FSC.Configuration.Month.DayOffsetHelp": "Le montant pour décaler le numéro du jour. Utilisé si vous voulez commencer un mois à 2 au lieu de 1",
+  "FSC.Configuration.Month.Number": "Numéro de mois",
+  "FSC.Configuration.Month.NumberHelp": "La représentation numérique de ce mois.",
+  "FSC.Configuration.Month.StartingWeekDay": "Démarre le jour de la semaine",
+  "FSC.Configuration.Month.StartingWeekDayHelp": "Choisissez par quel jour de la semaine ce mois commencera toujours, ou laissez par défaut pour laisser le calendrier déterminer le jour de la semaine approprié.",
+  "FSC.Configuration.Month.Remove": "Supprimer",
+  "FSC.Configuration.Month.RemoveAll": "Supprimer tous les mois",
+  "FSC.Configuration.Month.Add": "Ajouter un nouveau mois",
+  "FSC.Configuration.Months": "Mois",
+  "FSC.Configuration.Weekday.Title": "Paramètres des jours de la semaine",
+  "FSC.Configuration.Weekday.ShowHeadings": "Afficher les titres des jours de la semaine",
+  "FSC.Configuration.Weekday.ShowHeadingsHelp": "Si pour afficher les en-têtes des jours de la semaine sur le calendrier.",
+  "FSC.Configuration.Weekday.Name": "Nom du jour de la semaine",
+  "FSC.Configuration.Weekday.NameHelp": "Le nom textuel de ce jour de la semaine.",
+  "FSC.Configuration.Weekday.Add": "Ajouter un nouveau jour de la semaine",
+  "FSC.Configuration.Weekday.RemoveAll": "Supprimer tous les jours de la semaine",
+  "FSC.Configuration.Weekdays": "Jours de la semaine",
+  "FSC.Configuration.Weekday.Options": "Options des jours de la semaine",
+  "FSC.Configuration.Weekday.FirstWeekDay": "Démarrage du jour de la semaine",
+  "FSC.Configuration.Weekday.FirstWeekDayHelp": "Ceci définit le jour de la semaine où commence le premier jour de l'année 0.",
+  "FSC.Configuration.DefaultNoteVisibility": "Notez la visibilité par défaut du lecteur",
+  "FSC.Configuration.DefaultNoteVisibilityHint": "Pour les nouvelles notes, si par défaut l'option de visibilité du joueur est cochée ou non",
```

Comment:
This should be "Pour les nouvelles notes, la visibilité par défaut pour les joueurs.". The original string should probably be reworded too.